SECURITY.md

```diff
@@ -1,12 +1,12 @@
 ## Caveats
-Several caveats should be carefully considered before using PSI-Cardinality.
+Several caveats should be carefully considered before using PSI.
 
 ### Information assumed public 
 1. Server set size
 2. Client set size
 (Note that Each of these can be turned into upper bounds by adding dummy elements.)
 
-### Security Limitations for the PSI-Cardinality protocol
+### Security Limitations for the PSI protocol
 
 Coordinated clients could get the actual intersection. However, server set items not
```

Comment:
For PSI, clients getting the actual intersection is what we want. Maybe add something about the reveal_intersection flag here?